### PR TITLE
Log the forwarding rule associated with the federation APIServer service.

### PR DESCRIPTION
**Pull request by madhusudancs**
While deploying a federation control plane, we allocate a `type=LoadBalancer` service to Federation APIServer and poll until a loadbalancer address is assigned to this service. This polling times out sometimes and it is unclear what is going on here. So adding this additional debug snippets to gather additional information.

**Release note**:
```release-note
NONE
```

/assign @shashidharatd @csbell 

cc @nicksardo

### CODE REVIEW

File: federation/cluster/federation-up.sh [hunk 1]
```diff
@@ -113,6 +115,31 @@ function join_clusters() {
...
+  while ! svc_uid=$("${KUBE_ROOT}/cluster/kubectl.sh" \
+    --context="${HOST_CLUSTER_CONTEXT}" \
+    --namespace=${FEDERATION_NAMESPACE} \
+    -o jsonpath='{.metadata.uid}' \
+    get svc \
+    "${FEDERATION_NAME}-apiserver" 2>/dev/null); do
+    sleep 10
+  done
+
+  fwid_full="a${svc_uid//-/}"
```
Comment (shashidharatd) [inline federation/cluster/federation-up.sh]: is the prefix 'a' intentional?

Comment (madhusudancs) [inline federation/cluster/federation-up.sh]: Yeah. Based on this - https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/cloud.go#L62

**Comment by shashidharatd** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Pull request closed without merging**